clients/node/examples/2-UpdateUser.ts

```diff
@@ -1,38 +1,46 @@
 import { searchCriteria, IdentityClient, IdentityJson } from 'iota-is-sdk';
 import { defaultConfig } from './configuration';
 import { readFileSync } from 'fs';
-
+import _ from 'lodash';
 async function searchIdentityAndUpdate() {
-
   const identity = new IdentityClient(defaultConfig);
 
   // Recover the admin identity
-  const adminIdentity = JSON.parse(readFileSync("./adminIdentity.json").toString()) as IdentityJson;
+  const adminIdentity = JSON.parse(readFileSync('./adminIdentity.json').toString()) as IdentityJson;
```

Comment:
Do you want to put in this example: create identity and update identity instead?
In the way as it is written the example assumes that other identities exists and this can be a problem.
Make this test independant from other tests..

Reply:
Ok, I will create an identity in advance and then update this one.